src/main/java/seedu/address/logic/commands/AddEventCommand.java

```diff
@@ -18,7 +19,8 @@ public class AddEventCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates and adds an event to the address book. "
             + "Parameters: "
-            + PREFIX_EVENT_NAME + "EVENT_NAME";
+            + PREFIX_EVENT_NAME + "EVENT_NAME"
+            + PREFIX_DATE + "EVENT_DATE";
```

Comment:
its better if there are exmaples, check other command classes for reference

Reply:
Added examples.